.rubocop_todo.yml

```diff
@@ -57,7 +57,7 @@ Style/MixinUsage:
   Exclude:
     - 'lib/traject/psulib_config.rb'
 
-# Offense count: 57
+# Offense count: 59
```

Comment:
might want to add an exclude for `setting.yml` or turn that large array into a format like 

```yml
location_excludes:
  - item 1
  - item 2
...
```

small thing